src/core/config.ts

```diff
@@ -85,7 +85,9 @@ export class Config {
       WARTHOG_GENERATED_FOLDER: path.join(this.PROJECT_ROOT, 'generated'),
       WARTHOG_RESOLVERS_PATH: [path.join(this.PROJECT_ROOT, 'src/**/*.resolver.ts')],
       WARTHOG_SUBSCRIPTIONS: 'false',
-      WARTHOG_VALIDATE_RESOLVERS: 'false'
+      WARTHOG_VALIDATE_RESOLVERS: 'false',
+      WARTHOG_KEEPALIVE_TIMEOUT_MS: 30000,
+      WARTHOG_HEADER_TIMEOUT_OFFSET_MS: 5000
```

Comment:
Can we add `S` to header to match the server setting?

Reply:
Also, in new versions of Node this is set to 60 seconds: should we match that? https://github.com/nodejs/node/commit/e17403ede37d5879c2fa8e9d2c71f1eacbea9366